src/nectarchain/dqm/pixel_timeline.py

```diff
@@ -0,0 +1,155 @@
+from dqm_summary_processor import DQMSummary
+from matplotlib import pyplot as plt
+from ctapipe.visualization import CameraDisplay
+from ctapipe.instrument import CameraGeometry
+from ctapipe.coordinates import EngineeringCameraFrame
+import numpy as np
+
+
+class PixelTimeline_HighLowGain(DQMSummary):
+    def __init__(self, gaink):
+        self.k = gaink
+        return None
+
+    def ConfigureForRun(self, path, Pix, Samp, Reader1):
+        # define number of pixels and samples
+        self.Pix = Pix
+        self.Samp = Samp
+
+
+        self.counter_evt = 0
```

Comment:
Not PEP-8 compliant. Please enable `pre-commit` hooks before committing.

Reply:
done

Reply:
Well, actually I had to run another pass on `pixel_timeline.py` to lint it.